packaging/aznfs/DEBIAN/postinst

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Exit on error.
+set -e
+
+echo "===== postinst called. ====="
+
+chmod +x /sbin/aznfswatchdog
+chmod u+s,+x /sbin/mount.aznfs
```

Comment:
setuid doesn't work for scripts so this not needed.
Did you check if non-superuser is able to use this helper properly when an appropriate entry is added to /etc/fstab?
This is important!!